src/pages/SearchPage.js

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components/macro";
+import PropTypes from "prop-types";
+import ConcertCard from "../components/ConcertCard";
+import Heading from "../components/Header";
+
+SearchPage.propTypes = {
+  pageName: PropTypes.string,
+  concerts: PropTypes.array,
+  onClick: PropTypes.func,
```

Comment:
isRequired 🙂

Reply:
Sorry, that's not true, guess the case you want to use the button to submit a form, then you don't want to use the onClick. Instead you use the native onSubmit.

Reply:
but onNavigate is missing in the proptypes